package.json

```
@@ -6,7 +6,7 @@
   "author": "Pete West <peterjwest3@gmail.com> (https://github.com/peterjwest/)",
   "license": "MIT",
   "dependencies": {
-    "lodash": "~3.10.1"
+    "lodash": "~4.17.15"
```

As this is a library, designed to be incorporated into other projects, it would be good to keep the versions of lodash that are acceptable to as wide a range as possible, so the projects don't end up having to install multiple lodash versions. Unless you need something specific in a particular version, I'd use something like:

```suggestion
    "lodash": ">2"
```

(you can test the versions that will be installed with https://semver.npmjs.com/

---

I thought it was, but then this PR appeared...

---

Nothing to do with me! 😛